internal/cli/credstore/store.go

```diff
@@ -60,3 +65,15 @@ func DeleteHub(serverURL string) error {
 
 	return ks.Remove(b64.StdEncoding.EncodeToString([]byte(serverURL)))
 }
+
+func fileKeyringPassphrasePrompt(promptMessage string) (string, error) {
```

Comment:
probably, it can be some better place for this function but I would like to avoid creating ,,common" file. :)

Reply:
it's ok 👍